lib/email/store.rb

```diff
@@ -0,0 +1,38 @@
+# move this require in gemspec
+require 'dalli'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.